project-reports/2024/2024-Q2-Hyperledger-Identus.md

```diff
@@ -0,0 +1,62 @@
+
+
+# Questions/Issues for the TOC
+
+Is there a way to access funds for open source contirbutions and development? 
+
+
+# Releases
+
+  * Identus V2.12
+    * Identus Cloud Agent - V.1.33.0
+    * Identus Mediator - V.0.14.2
+    * Identus Apollo - V.1.2.14
+    * Identus Edge Agent SDK TS - V.5.1.0
+    * Identus Edge Agent SDK KMP - V.3.0.0
+    * Identus Edge Agent SDK SWIFT - V.6.0.0
+    * Identus Docs - V.1.92.0
+
+
+
+# Overall Activity in the Past Quarter
+
+The main focus this quarter, and continuing into the next, is to achieve a rich set of features within Identus. The contribution of the code base from IOG to Hyperledger was completed in early July 2024. Initially it was only the cloud agent respository that was part of Hyperledger that started out as the Open Enterprise Agent Lab. The rest of the repositories were public repositories in IOG and has since been migrated to Hyperledger. 
+
+The mailing list is not the primary tool for discussions, instead we encourage discussions and questions within the relevant Discord channels. To enhance support for community members, we have enlisted the CommunityOne bot, which is trained on the Identus documentation. This has been extremely helpful for developers seeking immediate answers when a contributor or maintainer is unavailable. The majority of development now is to get to feature complete and fix exisiting bugs that get reported by the community. 
+
+We currently utilize two Discord servers: the Hyperledger Discord and the Atala Discord. The Atala Discord has seen significantly more engagement. Since January 2024, our membership has grown from 258 to 476 members. On average, there are 348 activities across various channels monthly, with about 39 users online daily. We are currently encouraging people to utilize the Hyperledger Discord instead of the Atala Discord.
+
+
+# Current Plans
+
+As previously mentioned, the team will focus on enhancing the features of Identus. This entails enabling the mediator to become production ready, OIDC for Verifiable Credential Issuance (OID4VC), revocation mechanisms for JWT, Universal Resolver for Prism DIDs, Implementation of  user level events in SDKs and much more. This is constantly shared with the contributors and maintainers in order to come to a general consensus. 
+
+Community is also encouraged to log feature requests, issues etc and this is then incorporated into a roadmap that is transparent for everyone to see https://github.com/orgs/hyperledger/projects/48
+
+
+# Maintainer Diversity
+
+We currently have 24 maintainers with 2 maintainers from outside of IOG. We aim to expand the external maintainers list in the future
```

Comment:
Do you have specific plans for this? We have been doing some brainstorming at the TOC level on ways to help maintainers attract additional maintainers, and it would be great to share best practices.

Reply:
At the moment, we have not given it much thought as we focus on getting the additional features into Identus. The idea around funding would be one way along with technical workshops we want to host in the future. I also think that community contributed items such as the book (building with Identus) could be a nice way to encourage people to explore with the technology. 

Are there any specific ideas the TOC would recommend?

Reply:
We may want to put more guidance together with tips and suggestions on how to recruit more maintainers.  For now, I have this document with a range of community engagement suggestions that I recommend project maintainers do and this will help bring more people into a project and get them on the path to becoming maintainers.  Fay, I'm pretty sure you've seen this before but I wanted to post it here in case this is helpful as part of a document about how to recruit maintainers.

https://docs.google.com/presentation/d/13nji_R-op77ERT-AV3-CbOOZwAOjtvq33RRGnjpL3Gc/edit?usp=sharing